### ISSUE TITLE
Responses loaded through ARAX UI are missing original QNode IDs

### ISSUE THREAD

**Issue by colleenXu** [RTXteam/RTX#2027]
Pasted from Translator [Slack](https://ncatstranslator.slack.com/archives/C013Q5TVC87/p1683330761723379)

> We've noticed that the original QNode IDs in the QGraph are missing in responses that are loaded through ARAX. We think this is an ARAX issue because those IDs aren't missing in the original responses (seen on the ARS level) and this is affecting multiple ARAs.
> 
> - [BTE](https://github.com/biothings/biothings_explorer/issues/636#issuecomment-1535753424)
> - [Aragorn and ARAX](https://github.com/biothings/biothings_explorer/issues/636#issuecomment-1536908644)
> 
> This may be a recent issue...and we noticed it because we need to know the starting entities for a result-clustering/prioritization effort we are working on.

**Comment by isbluis** [comment]
Hi all,
This does not appear to be a UI issue, as the response that comes from the API  (that the UI uses) already has the ids missing, e.g.:
https://arax.ncats.io/api/arax/v1.3/response/2e756e1e-eaf8-4f90-95c7-f1f872c3b59b

![image](https://github.com/RTXteam/RTX/assets/37852256/0e6ae3b9-bb7f-407d-9a2b-12bf3c735011)

Could this be a validator-related issue, @edeutsch ?  Or ARS mangling?

**Comment by edeutsch** [comment]
You're right, that they seem to be missing from:
https://arax.ncats.io/api/arax/v1.3/response/2e756e1e-eaf8-4f90-95c7-f1f872c3b59b

In /beta, where I have turned off validation:
https://arax.ncats.io/beta/api/arax/v1.3/response/2e756e1e-eaf8-4f90-95c7-f1f872c3b59b
The ids are still there:
```
      "nodes": {
        "n0": {
          "categories": [
            "biolink:Disease"
          ],
          "constraints": [],
          "ids": [
            "MONDO:0004979"
          ],
          "is_set": false,
          "name": "asthma"
        },
        "n1": {
          "categories": [
            "biolink:Gene"
          ],
          "constraints": [],
          "is_set": false
        },
        "n2": {
          "categories": [
            "biolink:SmallMolecule"
          ],
          "constraints": [],
          "ids": [
            "CHEMBL.COMPOUND:CHEMBL1642"
          ],
          "is_set": false,
          "name": "IMATINIB MESYLATE"
        }
      }
```

I have not determined that it is the validator itself that is doing it, or some other problem.

In any case, it seems that if you use /beta, then it works fine:
https://arax.ncats.io/beta/?r=2e756e1e-eaf8-4f90-95c7-f1f872c3b59b
This code should be rolled out to production soon.

**Comment by colleenXu** [comment]
Noting this, which may be related: https://github.com/NCATSTranslator/reasoner-validator/releases/tag/v3.5.3

**Comment by edeutsch** [comment]
Almost certainly. This should now be fixed in latest beta versions. Perhaps still not in production, I'm not certain.

**Comment by edeutsch** [comment]
Confirmed that the validator was destructively removing nodes.
This is now fixed with the newer validator 3.5.3 and up.
closing.
